pkg/util/constants/constants.go

```diff
@@ -28,4 +28,6 @@ const (
 	PVProvisionerGCEPD  = "kubernetes.io/gce-pd"
 	PVProvisionerAWSEBS = "kubernetes.io/aws-ebs"
 	PVProvisionerNone   = "none"
+
+	HTTPServerListenAddr = "0.0.0.0:8080"
```

Comment:
I tried listening to `127.0.0.1` and it seems the pod never becomes ready. 
When the kubelet sends a probe to the HTTP endpoint it will send it on the pod IP. Listening to `0.0.0.0` means being reachable from the outside on the IP addresses of all network interfaces of the pod(including the pod IP).
Whereas `127.0.0.1` means listening to requests only directed to localhost from within the pod.
So I've kept it at `0.0.0.0`.